book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(date, book)
+    Rental.new(date, self, book)
+  end
```

Comment:
- [ ]  You should update this method so that you will have valid Rental with `date, person,` and `book` created. In this case, it is just` date, book (self)`, and `book`. For example:
```
  def add_rental(date, person)
    Rental.new(date, person, self )
  end
```

Reply:
@gtekle 
implemented the changes